portal-api/src/v1/services/utilisation-report/email-service.test.ts

```diff
@@ -53,12 +57,23 @@ describe('emailService', () => {
 
     it('should throw an error if email recipients not provided', async () => {
       // Arrange
+      process.env.TFM_UI_URL = 'https://www.ukexportfinance.gov.uk';
       delete process.env.UKEF_GEF_REPORTING_EMAIL_RECIPIENT;
 
       // Act + Assert
       await expect(sendUtilisationReportUploadNotificationEmailToUkefGefReportingTeam('My Bank', 'June 2026')).rejects.toThrow();
       expect(sendEmail).toHaveBeenCalledTimes(0);
     });
+
+    it('should throw an error if TFM_UI_URL is undefined', async () => {
+      // Arrange
+      delete process.env.TFM_UI_URL;
+      process.env.UKEF_GEF_REPORTING_EMAIL_RECIPIENT = '["email1@ukexportfinance.gov.uk"]';
+
+      // Act + Assert
+      await expect(sendUtilisationReportUploadNotificationEmailToUkefGefReportingTeam('My Bank', 'June 2026')).rejects.toThrow();
```

Comment:
Could we explicitly assert that the thrown error is the `InvalidEnvironmentVariableError` error?